### DISCUSSION TITLE
Allow to set avatar / profile picture

### DISCUSSION THREAD

**Discussion by mxmehl** [ente-io/ente#1205] [Enhancements]
It would be great if one could set an avatar/profile picture.

Background: I am using this great piece of software together with two other persons who are part of my family plan. However, all our three email addresses start with the same character which leads to the situation that I cannot recognise from the small round circle with this character from whom a picture is or with whom I've shared it. Only a colour indicates this, but I cannot always remember it.

Thanks for considering this feature!

**Comment by vishnukvmd** [comment]
Thanks for the feedback! Once we ship face recognition, we'll provide a way to attach a face to a contact.

**Comment by vishnukvmd** [comment]
This is now available: https://ente.io/blog/photos-v0.9.98/#faces-to-contacts
